environment.yml

```diff
@@ -5,11 +5,9 @@ channels:
   - fastai
 dependencies:
   - annotated-types
-  - bqplot
   - fastcore
   - immutables
   - ipydatagrid
-  - ipyfilechooser
```

Comment:
This one will get installed anyway because it is a dependency of ipyautoui, so no harm in dropping it here.